packages/web/src/components/pages/Tests/TestsList/TestCreationModalContent/TestCreationForm.tsx

```diff
@@ -135,7 +147,12 @@ const TestCreationForm: React.FC<TestCreationFormProps> = props => {
             ];
 
             return (
-              <FormItem rules={[required]} label="Source" name="testSource" required>
+              <FormItem
+                rules={!isContainerExecutor ? [required] : []}
```

Comment:
not important: could be reversed, but not a big deal:
```suggestion
                rules={isContainerExecutor ? [] : [required]}
```